### ISSUE TITLE
Yves Rocher wikidata

### ISSUE THREAD

**Issue by ivanbranco** [osmlab/name-suggestion-index#9981]
https://nsi.guide/index.html?t=brands&k=shop&v=cosmetics&tt=yves%20ro#yvesrocher-93ef6e

Shouldn't the wikidata be [Q1477321](https://www.wikidata.org/wiki/Q1477321) instead of  [Q28496595](https://www.wikidata.org/wiki/Q28496595)?

**Comment by Snowysauce** [comment]
I looked into this, and the situation turned out to be complicated. The page histories for both items ([Q1477321](https://www.wikidata.org/w/index.php?title=Q1477321&action=history&offset=&limit=500), [Q28496595](https://www.wikidata.org/w/index.php?title=Q28496595&action=history&offset=&limit=500)) are bizarre; at one point, Q1477321 referenced what is now the subject of Q28496595, and vice versa. It was during this period that the brand was added to the NSI, so the QID stored in the NSI was technically right at the time. This is a weird instance that admins at Wikidata really should look at, but from the standpoint of the NSI and this issue, yes, Q1477321 now looks to be the correct QID.
